config/locales/translation_ja.yml

```
@@ -0,0 +1,36 @@
+ja:
+  activerecord:
+    models:
+      book: 本  #g
+
+    attributes:
+      book:
+        author: 著者  #g
+        memo: メモ  #g
+        picture: 写真  #g
+        title: タイトル  #g
+
+  helper:
+    books:
+      common:
+        created: 本を作成しました
```

```suggestion
        created: %{name}を作成しました
```

というようにパラメータ化すると他にmodelが増えたときも再利用可能です。
（ただし、その場合はkey名にbooksを含まない方がいいですね）